data/src/main/kotlin/com/ortin/gesturetranslator/data/db/UserRoomDB.kt

```diff
@@ -0,0 +1,25 @@
+package com.ortin.gesturetranslator.data.db
+
+import android.content.Context
+import androidx.room.Database
+import androidx.room.Room
+import androidx.room.RoomDatabase
+import com.ortin.gesturetranslator.data.entities.UserEntity
+
+@Database(
+    entities = [UserEntity::class],
+    version = 1
+)
+abstract class UserRoomDB: RoomDatabase() {
+    abstract fun userDao(): UserDao
+
+    companion object {
+        fun createDataBase(context: Context): UserRoomDB {
```

Comment:
Здесь не создания базы данных. Ты в конце вызываешь метод build(). Поэтому функцию лучше назвать по-другому:

```suggestion
        fun buildDatabase(context: Context): UserRoomDB {
```

Reply:
done